packages/client/wallets/aa/src/blockchain/wallets/EVMAAWallet.ts

```diff
@@ -150,76 +283,15 @@ export class EVMAAWallet<B extends EVMBlockchainIncludingTestnet = EVMBlockchain
         }
     }
 
-    async setCustodianForKillswitch(custodian?: Custodian | undefined) {
```

Comment:
just checking–doing killswitch is still supported, but in a different way?

Reply:
is not supported in this version

Reply:
it's not supported in this version